src/main/java/com/exasol/projectkeeper/ProjectKeeperFitMojo.java

```diff
@@ -0,0 +1,25 @@
+package com.exasol.projectkeeper;
+
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.project.MavenProject;
+
+/**
+ * Entry point for the fit goal.
+ * <p>
+ * Run using {@code mvn project-keeper:fit}
+ * </p>
+ */
+@Mojo(name = "fit")
+public class ProjectKeeperFitMojo extends AbstractProjectKeeperMojo {
+
+    @Parameter(defaultValue = "${project}", required = true, readonly = true)
+    MavenProject project;
```

Comment:
Is it supposed to have a default access modifier?

Reply:
:+1: can be private